Sources/ArcGISToolkit/Components/Offline/PreplannedMetadataView.swift

```diff
@@ -0,0 +1,118 @@
+
+struct PreplannedMetadataView: View {
+    /// The view model for the preplanned map.
+    @ObservedObject var model: PreplannedMapModel
+    
+    /// The action to dismiss the view.
+    @Environment(\.dismiss) private var dismiss
+    
+    var body: some View {
+        Form {
+            Section {
+                if let image = model.preplannedMapArea.thumbnail {
+                    HStack {
+                        Spacer()
+                        LoadableImageView(loadableImage: image)
+                            .clipShape(.rect(cornerRadius: 10))
+                            .padding(.vertical, 10)
+                        Spacer()
+                    }
+                }
+                VStack(alignment: .leading) {
+                    Text("Name")
+                        .font(.caption)
+                        .foregroundStyle(.secondary)
+                    Text(model.preplannedMapArea.title)
+                        .font(.subheadline)
+                }
+                VStack(alignment: .leading) {
+                    Text("Description")
+                        .font(.caption)
+                        .foregroundStyle(.secondary)
+                    if model.preplannedMapArea.description.isEmpty {
+                        Text("This area has no description.")
+                            .font(.subheadline)
+                            .foregroundStyle(.tertiary)
+                    } else {
+                        Text(model.preplannedMapArea.description)
+                            .font(.subheadline)
+                    }
+                }
+                VStack(alignment: .leading) {
+                    Text("Size")
+                        .font(.caption)
+                        .foregroundStyle(.secondary)
+                    Text(model.directorySize.formatted(.byteCount(style: .file, allowedUnits: [.kb, .mb])))
+                        .font(.subheadline)
+                }
+            }
+            Section {
+                HStack {
+                    Image(systemName: "trash.circle.fill")
+                        .symbolRenderingMode(.palette)
+                        .foregroundStyle(.red, .gray.opacity(0.1))
+                        .font(.title)
+                    Button("Delete Map Area", role: .destructive) {
+                        dismiss()
+                        model.removeDownloadedPreplannedMapArea()
+                        Task {
+                            await model.load()
+                        }
+                    }
+                }
+            }
+        }
+        .navigationTitle(model.preplannedMapArea.title)
+        .navigationBarTitleDisplayMode(.inline)
+        .navigationBarBackButtonHidden()
+        .toolbar {
+            ToolbarItem(placement: .confirmationAction) {
+                Button {
+                    dismiss()
+                } label: {
+                    Text("Done")
+                        .bold()
+                }
```

Comment:
I'm fairly certain that SwiftUI makes the button text bold because of its placement. I don't see a noticeable different with and without the `bold()` modifier.

```suggestion
                Button("Done") {
                    dismiss()
                }
```

By the way, `Button("Done") { ... }` is used in three different places now and they will all need to be localized. I would suggest extending `Button` with a helper method to make that easier:

```swift
extension Button {
    static func done(action: @escaping () -> Void) -> Button where Label == Text {
        return .init(action: action) {
            Text(
                "Done",
                bundle: .toolkitModule,
                comment: "A label for a button for the user to indicate completion of the current task."
            )
        }
    }
}
```